src/assets/stylesheets/components/_input.scss

```
@@ -0,0 +1,37 @@
+.input-container {
+  position: relative;
+  padding-bottom: rem(24px);
+
+  .input-label {
+    display: flex;
+    flex: none;
+    flex-grow: 0;
+    align-items: center;
+    align-self: stretch;
+    order: 0;
+    width: 20rem;
+    height: 1.25rem;
+    margin-bottom: 0.5rem;
+
+    color: #fff;
+    font-family: $font-family-sans-serif-normal;
+    font-weight: 800;
+    letter-spacing: rem(-0.24px);
+  }
+
+  .input {
+    width: 100%;
+    height: 3.5rem;
+    padding-top: rem(19px);
+    padding-bottom: rem(15px);
+    padding-left: rem(12px);
```

We can use the shorthand for this here 👍 

```suggestion
    padding: rem(19px 12px 15px);
```

---

Changed in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/15f355d38785ddd24455c4372feb7a980237385b 👍